network/src/proto/consensus.proto

```diff
@@ -6,16 +6,15 @@ syntax = "proto3";
 package consensus;
 
 import "ledger_info.proto";
-import "transaction.proto";
-import "validator_set.proto";
 
 message ConsensusMsg {
   oneof message {
     Proposal proposal = 1;
     VoteMsg vote_msg = 2;
     RequestBlock request_block = 3;
     RespondBlock respond_block = 4;
-    SyncInfo sync_info = 6;
+    SyncInfo sync_info = 5;
```

Comment:
Wouldn't a sync info also have a HLI? Could a sync info serve this role?

Reply:
it could, I feel a different message type is more explicit about the purpose. alternatively we can have a special logic when processing sync_info, I don't have strong opinions here.